Core/Util/Configs/Components/ConfigPlayer.cs

```diff
@@ -15,4 +16,44 @@ public class ConfigPlayer: ConfigElement<ConfigPlayer>
     [ConfigInfo("Gender of the player.")]
```

Comment:
I think this add warrants moving the player config to it's own page. This is easily done by adding Player to the OptionsSectionType enum after the General and changing the OptionMenu attribute to use OptionsSectionType.Player

You can also add spacing to the groups:
[OptionMenu(OptionSectionType.Player, "Group 1 1st", spacer: true)]

Reply:
Moved to a new page and added the spacing.